.github/workflows/ci.yml

```diff
@@ -54,39 +54,52 @@ jobs:
     - name: Build all projects
       run: ./scripts/build.sh ${{ env.CI_VERSION }} Release
     
-    - name: Test
+    - name: Test (.NET - TRX)
       run: dotnet test --configuration Release --verbosity normal --logger "trx;LogFileName=test-results.trx" --results-directory ./TestResults
     
-    - name: Run cycodt tests
+    - name: Run cycodt tests (JUnit)
       run: |
         export PATH=$PATH:$(pwd)/src/cycod/bin/Release/net9.0:$(pwd)/src/cycodt/bin/Release/net9.0:$(pwd)/src/cycodmd/bin/Release/net9.0
         which cycod
         which cycodmd
         which cycodt
-        cycodt run --log ./TestResults/test-results-cycodt.log --output-file ./TestResults/test-results-cycodt.trx
+        cycodt run --log ./TestResults/test-results-cycodt.log --output-file ./TestResults/test-results-cycodt.xml --output-format junit
     
-    - name: Clean up inception test TRX files
+    - name: Clean up inception test result files
       run: |
-        # Remove TRX files created by inception tests to avoid interference with CI reporting
-        rm -f tests/cycodt-yaml/inception-layer-1/*.trx
-        echo "Cleaned up inception test TRX files"
+        # Remove TRX files created by inception tests (still may exist) and any stray trx
+        rm -f tests/cycodt-yaml/inception-layer-1/*.trx || true
+        echo "Cleaned up inception test legacy TRX files"
     
+    - name: Convert TRX to JUnit XML
```

Comment:
This isn't required for cycodt yaml tests, yes? as we're outputing in junit directly, correct? if so, then, i'd suspect we don't need this for the TRX from dotnet test, as that doesn't seem to be the one that was having difficulty uploading, right? i'm unsure, but I'd like to check first... it's a bit non-standard to have .net tests uploaded in junit format... and, if we want that, we can probably just ask dotnet test to output in that format... and not have this step of the process....